modules/core/src/main/java/org/apache/ignite/internal/management/cdc/CdcDeleteLostSegmentsTask.java

```diff
@@ -102,69 +115,138 @@ protected CdcDeleteLostSegmentsJob(CdcDeleteLostSegmentLinksCommandArg arg, bool
             try {
                 lock.tryLock(1);
 
-                try (Stream<Path> cdcFiles = Files.list(walCdcDir.toPath())) {
-                    Set<File> delete = new HashSet<>();
+                Long lostSgmnt = findLastLostSegement();
 
-                    AtomicLong lastSgmnt = new AtomicLong(-1);
+                if (lostSgmnt != null)
```

Comment:
if (seg != null) -> this check can be moved to the method.
It seem, that will simplify the code.